### PR TITLE
Responsive BottomSheet

### PR DESCRIPTION
**Pull request by tommypoa**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Replaced max snap point of the BottomSheet to be responsive to the height of the device
- Not entirely pixel perfect (not sure what the repercussions of using percentages is as compared to the other option which is `pts`)

## Relevant Links
- NA

[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

## How to review

[//]: # "The order in which to review files and what to expect when testing locally"
- Responsiveness to different screens 

## Next steps

[//]: # "What's NOT in this PR, doesn't work yet, and/or still needs to be done"

## Tests Performed, Edge Cases

[//]: # "Hopefully we will add a testing suite/CI soon, but until then note down the steps you took to test locally"
- Checked responsiveness for devices of different screen sizes
### Screenshots

[//]: # "Add screenshots of expected behavior - GIFs if you're feeling fancy!"

CC: @wangannie

[//]: # "This tags in both Annies as a default. Feel free to change, or add on anyone who you should be in on the conversation."

### CODE REVIEW

File: screens/map/MapScreen.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ import {
   SearchBar,
 } from '../../styled/store';
 
+const minSnapPoint = 300;
```
Comment (wangannie) [inline screens/map/MapScreen.js:23]: Also this should be a little higher. In Figma you should be able to peek the images https://www.figma.com/file/drhxZht9Mvogrc6aLn7MsK/Spring-20-DC-Central-Kitchen?node-id=7827%3A55

[the remark this answers is not in the document]
Comment (tommypoa) [reply screens/map/MapScreen.js:23]: I was going to, but the math does not really add up and was done largely by trial and error. Should I still have it down?

Comment (tommypoa) [reply screens/map/MapScreen.js:23]: Changed `minSnapPoint` to `325`, which is a little higher (16px vs 11px in figma) but given that our current product cards do not have backgrounds, I gave it a little bump to show at least some parts of the product image. Feel free to change it to something that makes more sense in terms of design decision!

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,31 +1,25 @@
...
-    "editor.formatOnSave": true,
-    "editor.codeActionsOnSave": {
-        "source.organizeImports": true,
-        "source.fixAll.eslint": true
-    },
-    "javascript.suggest.completeFunctionCalls": true,
-    "javascript.implicitProjectConfig.checkJs": true,
-}
\ No newline at end of file
+  "files.associations": {
```
[the remark this answers is not in the document]
Comment (tommypoa) [reply .vscode/settings.json:2]: This is strange - even though I inevitably caused these changes due to format on save (no way to turn it off somehow) but what I've changed it to is what is currently reflected in master so I'm hoping..that might be ok?

**Comment by wangannie** [comment]
Added minimum snap point for map accessibility on really small phones like the iPhone SE.
Min
